src/pudl/glue/ferc1_eia.py

```diff
@@ -452,6 +452,25 @@ def get_db_utils_eia(pudl_engine):
     return db_utils_eia
 
 
+def get_utility_most_recent_capacity(pudl_engine):
+    """Get a list of all utilities' most recent total net capacity of their generators."""
```

Comment:
It might not make sense to only look at the most recent capacities to rank the utilities, since we're sometimes mapping older entities.  But I guess at this point usually it's a new year of data, so maybe it doesn't matter.